config/create_genesis.ts

```diff
@@ -0,0 +1,180 @@
+import { BigNumber, ethers } from "ethers";
+import { exec } from "child_process";
+import { glob } from "glob";
+import { keccak256 } from "ethereumjs-util";
+import hre from "hardhat";
+import fs from "fs";
+import path from "path";
+import util from "node:util";
+
+type PreDeploy = {
+  address: string;
+  balance: string;
+  contract: string | undefined;
+  storage: { [key: string]: string };
+};
+
+async function main() {
+  const { baseGenesisPath, genesisPath } = parseArgs();
+  await generateGenesisFile(baseGenesisPath, genesisPath);
+}
+
+export async function generateGenesisFile(
```

Comment:
nit: some brief documentation for each of these functions would be helpful

Reply:
added some JSDOC to these functions